examples/html2text.rs

```diff
@@ -7,88 +7,84 @@ use log::trace;
 use std::io;
 use std::io::Write;
 
-#[cfg(unix)]
 use html2text::render::RichAnnotation;
-#[cfg(unix)]
 fn default_colour_map(
     annotations: &[RichAnnotation],
     s: &str,
     use_css_colours: bool,
     no_default_colours: bool,
 ) -> String {
-    use termion::color::*;
+    use yansi::{hyperlink::HyperlinkExt, Paint};
     use RichAnnotation::*;
     // Explicit CSS colours override any other colours
     let mut have_explicit_colour = no_default_colours;
-    let mut start = Vec::new();
-    let mut finish = Vec::new();
+    let mut styled = s.to_string();
     trace!("default_colour_map: str={s}, annotations={annotations:?}");
     for annotation in annotations.iter() {
-        match annotation {
-            Default => {}
-            Link(_) => {
-                start.push(format!("{}", termion::style::Underline));
-                finish.push(format!("{}", termion::style::Reset));
-            }
-            Image(_) => {
+        styled = match annotation {
+            Default => styled,
+            Link(url) => styled.link(url).blue().underline().to_string(),
+            Image(img) => {
                 if !have_explicit_colour {
-                    start.push(format!("{}", Fg(Blue)));
-                    finish.push(format!("{}", Fg(Reset)));
+                    format!(
+                        "{} {}",
+                        styled.yellow().italic(),
+                        "img".underline().blue().link(img)
+                    )
+                } else {
+                    styled
                 }
             }
-            Emphasis => {
-                start.push(format!("{}", termion::style::Bold));
-                finish.push(format!("{}", termion::style::Reset));
-            }
+            Emphasis => styled.italic().to_string(),
             Strong => {
                 if !have_explicit_colour {
-                    start.push(format!("{}", Fg(LightYellow)));
-                    finish.push(format!("{}", Fg(Reset)));
+                    styled.bold().to_string()
+                } else {
+                    styled
                 }
             }
             Strikeout => {
                 if !have_explicit_colour {
-                    start.push(format!("{}", Fg(LightBlack)));
-                    finish.push(format!("{}", Fg(Reset)));
+                    styled.strike().to_string()
```

Comment:
The text should already be struck out already by this point (using [U+336](https://www.unicodepedia.com/unicode/combining-diacritical-marks/336/combining-long-stroke-overlay/)).